llvm/lib/Target/RISCV/MCTargetDesc/RISCVBaseInfo.cpp

```diff
@@ -130,6 +130,118 @@ parseFeatureBits(bool IsRV64, const FeatureBitset &FeatureBits) {
 
 } // namespace RISCVFeatures
 
+bool RISCVII::vectorInstUsesNBitsOfScalarOp(uint16_t Opcode, unsigned Bits,
```

Comment:
This interface makes more sense if it take Opcode and SEW and returns the number of bits consumed, and the comparison is done by the caller.  This is overly specific to the current caller.